packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -642,11 +502,94 @@ export class Relayer {
         } else throw receipt;
       } else throw receipt;
     } catch (error) {
-      this.logger.error({ at: "InsuredBridgeRelayer#Disputer", message: "Something errored disputing relay!", error });
+      this.logger.error({ at: "AcrossRelayer#Disputer", message: "Something errored disputing relay!", error });
     }
   }
 
-  private generateSlowRelayTx(deposit: Deposit, realizedLpFeePct: BN): TransactionType {
+  private async _processTransactionBatch(
+    transactions: { transaction: TransactionType | any; message: string; mrkdwn: string }[]
+  ) {
+    // Remove any undefined transaction objects or objects that contain null transactions.
+    transactions = transactions.filter((transaction) => {
+      if (!transaction) return false;
```

Comment:
why not do `return transaction && transaction.transaction`

Reply:
sure.